shesmu-server/src/main/java/ca/on/oicr/gsi/shesmu/Server.java

```diff
@@ -260,6 +260,24 @@ public Server(int port, FileWatcher fileWatcher) throws IOException, ParseExcept
             fileName ->
                 new GuidedMeditation(
                     fileName, DefinitionRepository.concat(definitionRepository, compiler)));
+
+    /**
+     *
```

Comment:
How do I get the formatter to not put these two lines above a <pre> tag? if I remove them, it puts them back on commit.

Reply:
Is there a particular reason that you want the two lines removed?

Reply:
You could see if there's a config setting to remove those two lines, but the formatter is pretty opinionated so that toggle might not exist.